automation/taskcluster/decision_task.py

```diff
@@ -193,7 +193,7 @@ def linux_build_task(name):
         })
         .with_index_and_artifacts_expire_in(build_artifacts_expire_in)
         .with_artifacts("/build/sccache.log")
-        .with_max_run_time_minutes(60)
+        .with_max_run_time_minutes(120)
```

Comment:
Really?  Is `x86_64` particularly slow?

Reply:
I could have been unlucky, but yeah the job timed out on me :/